workers/requirements.in

```diff
@@ -0,0 +1,9 @@
+amqp
```

Comment:
:+1:  So this is the same method you are using elsewhere in this project, right (having a requirements.in file and then running pip-compile)?

Reply:
By the way, is there anywhere else in the project where we need to switch to this method of dealing with requirements (for the sake of standardization)?

Reply:
Indeed :100:

Reply:
Ah, did you see the second comment? Sometimes Github doesn't refresh.

Reply:
Nope, this was the last holdout :)

Reply:
Nice. :+1: